tests/test_pytorch/test_accelerator.py

```diff
@@ -32,6 +32,20 @@
 from lightning_habana.pytorch.strategies import HPUParallelStrategy, SingleHPUStrategy
 from tests.helpers import ClassifDataModule, ClassificationModel
 
+# The external strategies, accelerator gets registerd with trainer initialization
+try:
+    if module_available("lightning"):
+        from lightning.pytorch.accelerators import HPUAccelerator as HPUAccelerator_Registered
+        from lightning.pytorch.strategies import HPUParallelStrategy as HPUParallelStrategy_Registered
+        from lightning.pytorch.strategies import SingleHPUStrategy as SingleHPUStrategy_Registered
+    elif module_available("pytorch_lightning"):
+        from pytorch_lightning.accelerators import HPUAccelerator as HPUAccelerator_Registered
+        from pytorch_lightning.strategies import HPUParallelStrategy as HPUParallelStrategy_Registered
+        from pytorch_lightning.strategies import SingleHPUStrategy as SingleHPUStrategy_Registered
+except Exception:
+    print("Trainer not initialized, HPU registration not complete")
+    pass
```

Comment:
```suggestion
if module_available("lightning"):
    from lightning.pytorch.accelerators import HPUAccelerator as HPUAccelerator_Registered
    from lightning.pytorch.strategies import HPUParallelStrategy as HPUParallelStrategy_Registered
    from lightning.pytorch.strategies import SingleHPUStrategy as SingleHPUStrategy_Registered
elif module_available("pytorch_lightning"):
    from pytorch_lightning.accelerators import HPUAccelerator as HPUAccelerator_Registered
    from pytorch_lightning.strategies import HPUParallelStrategy as HPUParallelStrategy_Registered
    from pytorch_lightning.strategies import SingleHPUStrategy as SingleHPUStrategy_Registered
```
Why does this need to be in try/except?

Reply:
This depends on lightning's trainer initialization and am not sure what will be the future chance. Hence its within a try catch block

Reply:
lets rather have it without and edit tests if needed later